tools/scan-images.sh

```diff
@@ -0,0 +1,72 @@
+set -eo pipefail
+
+# Check correct usage
+if [[ ! $2 ]]; then
+    echo "Usage: overcloud-ubuntu-upgrade.sh <os-distribution> <image-tag>"
+    exit 2
+fi
+
+set -u
+
+# Check that trivy is installed
+if ! trivy --version; then
+    echo 'Please install trivy: curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.49.1'
+fi
+
+# Make a fresh output directory
```

Comment:
If there is an existing directory we should either remove it or fail

Reply:
Done